src/app/filter/filter-fields.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="input-group form-group">
     <div class="input-group-btn" dropdown>
       <button type="button" class="btn btn-default filter-fields dropdown-toggle" dropdownToggle
-              tooltip="Filter by" placement="{{config?.tooltipPlacement}}">
+              tooltip="Filter&nbsp;by" placement="{{config?.tooltipPlacement}}">
```

Comment:
why not apply `white-space: nowrap` to the button instead?

https://developer.mozilla.org/en-US/docs/Web/CSS/white-space